components/brave_wallet/browser/brave_wallet_service.cc

```diff
@@ -1281,7 +1281,7 @@ void BraveWalletService::Base58Encode(
 }
 
 void BraveWalletService::DiscoverAssetsOnAllSupportedChains() {
-  DiscoverAssetsOnAllSupportedChains(false);
+  DiscoverAssetsOnAllSupportedChains(true);
```

Comment:
This is also called on `WalletActions.refreshAll` and now will bypass rate_limit which is not what we might want

Reply:
Updated. PTAL